### DISCUSSION TITLE
R Bootcamp in Quarto

### DISCUSSION THREAD

**Discussion by reuning** [quarto-dev/quarto-cli#4569] [Show and Tell]
Hey all, 

Just wanted to thank you for putting this together. I am about to finish teaching an [R Bootcamp and decided to try putting the slides together in Quarto and RevealJS](https://kevinreuning.com/presentations/). I think the ability to easily copy code and try it helped, also the ability to step through code. 

For the last day I'm actually showing them a little Quarto as well.

Thanks again, 

Kevin

**Comment by LeonorWhetsellnd** [comment]
Kevin! It's great to hear that you found Quarto and RevealJS helpful in your teaching. Quarto's ability to easily integrate code and visualizations into documents can certainly make teaching programming concepts more interactive and engaging for learners. Good luck with the rest of your bootcamp!
